src/components/application_manager/include/application_manager/plugin_manager/rpc_plugin.h

```diff
@@ -65,7 +65,8 @@ enum ApplicationEvent {
   kApplicationRegistered,
   kApplicationUnregistered,
   kDeleteApplicationData,
-  kGlobalPropertiesUpdated
+  kGlobalPropertiesUpdated,
+  kRCStatusChanged
```

Comment:
Smells like a separate bug, please mention in description this changes and do separate commit